### DISCUSSION TITLE
Keep Monthly Image on x day of month for bill/actual submitting etc

### DISCUSSION THREAD

**Discussion by jtricerolph** [jomjol/AI-on-the-edge-device#1822] [Ideas] [answered]
In the current situation people are even more likely to be routinely submitting readings for non smart meters or have issues with price changes etc.
Would be a nice feature to keep a longer period of reading images for a set interval for submitting or proof should there be issues with a supplier.
Something like maybe daily for x days,
Weekly for x weeks on x day of week
Monthly for x months on x day of month
Couple that with positioning of cam to make sure serial is in frame and could prove valuable when bill needs challenging.

**Comment by caco3** [comment, marked as answer]
This is currently not implemented, although could be implemented if somebody wants to do it.

How ever it might be easier to collect the image with an external tool.

The [Demo Mode documentation](https://jomjol.github.io/AI-on-the-edge-device-docs/Demo-Mode/#recording) shows an example hot to collect it remotely

**Reply by jtricerolph** [reply]
That was the path I was planning to take, have set the image as a generic cam in home assistant to make it viable at anytime local or remote on dashboard and will whip up HA to grab a copy on a schedule.
Just thought might be worth suggesting, with energy the way it is at the minute people will want to ensure acurate readings can be provided and proving a reading in case of challenging a bill issue would be easier with an image.
